geomesa-geojson/geomesa-geojson-api/src/test/scala/org/locationtech/geomesa/geojson/GeoJsonGtIndexTest.scala

```diff
@@ -46,15 +46,15 @@ class GeoJsonGtIndexTest extends Specification {
       result must haveLength(3)
       result must contain(f0, f1, f2)
 
-      val nameResult = index.query(name, """{ "properties.name" : "n1" }""").toList
+      val nameResult = index.query(name, """{ "$.properties.name" : "n1" }""").toList
```

Comment:
do we have to add a `$.` to attributes now? i'd like to not change the geojson-api syntax if possible